### PR TITLE
Milestone 2: Works and About Myself

### PR DESCRIPTION
**Pull request by cnualartc**
- created cards using grid layout
- aligned images and content on cards using flexbox
- created categories using unordered lists.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -73,3 +78,202 @@ header {
...
+button {
+  padding: 10px;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-size: 25px;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 10px;
+}
+
```
Comment (zieeco) [inline style.css]: - Hey, please, let's reduce the font size to 17px to match the design as much as we can from Figma
- Let's also add the interacting effect of the button `hover, disabled, active` states to improve the user experience when interacting with the button below:

*figma*
Screenshot from 2023-01-14 01-35-32

*your design*
Screenshot from 2023-01-14 01-35-21

---

*button actions*


Screenshot from 2023-01-14 01-40-31

File: index.html [hunk 2]
```diff
@@ -53,6 +53,315 @@ <h1>I'm Christian <br>
...
+              <a href="">Frameworks</a>
+            </div>
+          </div>
+        </li>
+        <li class="language-name">
+          <div class="positioning">
+            <div>
+              <a href="">Skills</a>
+            </div>
+          </div>
```
Comment (zieeco) [inline index.html]: - Hey, please, kindly add the missing dropDown image close to the framework and skills to match the design as much as you can below:

*Figma design*
Screenshot from 2023-01-14 01-43-05

*Your design*
Screenshot from 2023-01-14 01-43-32

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @cnualartc ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (cnualartc) [reply index.html]: @zieeco Thank you so much for the feedback 👍🏼
I made the changes and I am ready to request a review.

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @cnualartc 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good implementing feedback ✅ 
- Good use of GitHub flow ✅ 
- No linter error ✅ 

## Optional suggestions
- I think you can still improve the quality of your ReadMe by strictly updating it in accordance with descriptive [README template](https://github.com/microverseinc/readme-template), and please remember to customize it to your project.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**